Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'turbolinks'
 
 # View
 gem 'simple_form'
+gem 'enumerize'
```

Comment:
So apparently I'm an idiot (or just my multitasking skills are failing me). As @cawel pointed out the `enumerize` gem was moved, not added -- I missed the deletion for some reason.